.github/workflows/spec.yml

```diff
@@ -42,7 +42,11 @@ jobs:
           bundle exec rake db:setup --trace
           bundle exec bin/rails webdrivers:chromedriver:update
       - name: spec/${{ matrix.suite }}
-        run: bundle exec rake spec:${{ matrix.suite }}
+        if: matrix.suite != 'leftovers'
+        run: bundle exec rspec spec/${{ matrix.suite }}
+      - name: spec/leftovers
+        if: matrix.suite == 'leftovers'
+        run: bundle exec rspec --exclude-pattern "spec/{models, features, controllers, ability}/**/*_spec.rb, spec/datatables/**/*_spec.rb"
```

Comment:
I do want to eventually look into the datatables issues, but that is for a separate PR. That being said, we should pend the datatables specs within the spec files so as to show the tests and pending instead of excluding them via a pattern match.

Reply:
That makes sense. I'll change the tests then